Update docs (#171)
JuliaSmoothOptimizers/OptimizationProblems.jl: docs/src/tutorial.md

## Problems in JuMP syntax: PureJuMP

> You can obtain the list of problems currently defined with `OptimizationProblems.meta[!, :name]` that returns the list of exported names of the submodule.

Reviewer: ```suggestion
You can obtain the list of problems currently defined with `OptimizationProblems.meta[!, :name]`.
```

``` @example ex1
using OptimizationProblems, OptimizationProblems.PureJuMP
problems = OptimizationProblems.meta[!, :name]
length(problems)
```
Then, it suffices to select any of this problem to get the JuMP model.
``` @example ex1
jump_model = zangwil3()
```

> Note that certain problems are scalable, i.e., their size depends on parameters that can be modified. The list of those problems is available once again using `meta`.

Reviewer: ```suggestion
Note that certain problems are scalable, i.e., their size depends on parameters that can be modified. The list of those problems is available once again using `meta`:
```

> Note that some of these problems are scalable, i.e., their size depends on some parameters that can be modified. The list is available once again using the `meta`.

Reviewer: ```suggestion
Note that certain problems are scalable, i.e., their size depends on parameters that can be modified. The list of those problems is available once again using `meta`.
```

``` @example ex2
using ADNLPModels
```

> You can obtain the list of problems currently defined with `OptimizationProblems.meta[!, :name]`.

Reviewer: ```suggestion
You can obtain the list of problems currently defined with `OptimizationProblems.meta[!, :name]`:
```

> You can obtain the list of problems currently defined with `OptimizationProblems.meta[!, :name]` that returns the list of exported names of the submodule.

Reviewer: ```suggestion
You can obtain the list of problems currently defined with `OptimizationProblems.meta[!, :name]`.
```
